moveit_kinematics/ikfast_kinematics_plugin/templates/ikfast61_moveit_plugin_template.cpp

```diff
@@ -955,7 +1084,24 @@ bool IKFastKinematicsPlugin::searchPositionIK(const geometry_msgs::Pose& ik_pose
   // Initialize
 
   KDL::Frame frame;
-  tf::poseMsgToKDL(ik_pose, frame);
+  if (transform_tip_ || transform_base_)
```

Comment:
add a comment explaining this code section

Reply:
done